src/xcoobee/sdk/Users.js

```diff
@@ -203,8 +203,12 @@ class Users {
    *
    * @async
    * @param {string} message
-   * @param {*} consentId
-   * @param {*} breachId
+   * @param {Reference} reference - at least one of following keys should be provided
+   * @property {string} consentId
+   * @property {string} breachId

```

Comment:
In `ConversationsApi.sendUserMessage`, I only see `consentId`, `requestRef`, and `ticketId` being used. Should `breachId` and `complaintId` be removed from this doc comment?